erpnext/healthcare/doctype/lab_test/lab_test.py

```diff
@@ -225,16 +231,17 @@ def create_sample_doc(template, patient, invoice, company = None):
 			sample_collection.company = company
 
 			if(template.sample_details):
-				sample_collection.sample_details = "Test :" + (template.get("lab_test_name") or template.get("template")) +"\n"+"Collection Detials:\n\t"+template.sample_details
+				sample_collection.sample_details = 'Test :' + (template.get('lab_test_name') or template.get('template')) + '\n' + 'Collection Detials:\n\t' + template.sample_details
 			sample_collection.save(ignore_permissions=True)
 
 		return sample_collection
 
 def create_sample_collection(lab_test, template, patient, invoice):
-	if(frappe.db.get_value("Healthcare Settings", None, "create_sample_collection_for_lab_test") == "1"):
+	if(frappe.db.get_value('Healthcare Settings', None, 'create_sample_collection_for_lab_test') == '1'):
 		sample_collection = create_sample_doc(template, patient, invoice, lab_test.company)
 		if(sample_collection):
```

Comment:
```suggestion
		if sample_collection:
```
Remove brackets from all such conditions